test/utils/trends/data.test.js

```diff
@@ -49,12 +49,12 @@ describe('[trends] data utils', () => {
         expect(utils.findTimeOfDayBin(binSize, 0)).to.equal(1800000);
       });
 
-      it('should assign a bin of `1800000` to a datum at time 3600000', () => {
-        expect(utils.findTimeOfDayBin(binSize, 3600000)).to.equal(1800000);
+      it('should assign a bin of `1800000` to a datum at time 3599999', () => {
+        expect(utils.findTimeOfDayBin(binSize, 3599999)).to.equal(1800000);
```

Comment:
OK, I've made that change, but beware it will break things on every other branch we have in flight (where it's used for the smbg side of things in `SMBGRangeAvgAnimationContainer`, which I can't update from this branch which doesn't have that code yet).